.github/workflows/gemini-scheduled-issue-triage.yml

```diff
@@ -84,7 +84,7 @@ jobs:
             - Triage each issue independently based on title and body content
             - Focus on applying: kind/* (bug/enhancement/documentation), area/* (core/cli/testing/windows), and priority/* labels
             - If an issue has insufficient information, consider applying "status/need-information"
-            - After applying appropriate labels to an issue, remove the "status/need-triage" label if present: `gh issue edit ISSUE_NUMBER --repo ${{ github.repository }} --remove-label "status/need-triage"`
+            - After applying appropriate labels to an issue, add the "status/ready-for-human-review" label: `gh issue edit ISSUE_NUMBER --repo ${{ github.repository }} --add-label "status/ready-for-human-review"`
```

Comment:
with this change, the issue will have both `status/need-triage` and `status/ready-for-human-review` --> i recommend that the bot should remove `status/need-triage` and add `status/ready-for-human-review`